src/types/TableCell.d.ts

```diff
@@ -0,0 +1,29 @@
+interface Props {
+  /**
+   * Component's HTML Element
+   *
+   * @default 'td'
+   */
+  component?: 'td' | 'th';
```

Comment:
According to component source code there is no such component property. Why is it needed here we have `header` property. Or keep it here and add to component where if `component` is specified, then ignore `header` prop. Or remove `component` property from all Table-related components